stdlib/3/multiprocessing/connection.pyi

```diff
@@ -1,36 +1,61 @@
-from typing import Any, Iterable, List, Optional, Tuple, Type, Union
 import socket
 import types
+from multiprocessing.connection import Listener
+from typing import Any, Iterable, List, Optional, Tuple, Type, Union
 
 # https://docs.python.org/3/library/multiprocessing.html#address-formats
 _Address = Union[str, Tuple[str, int]]
 
+class _ConnectionBase:
+    @property
+    def closed(self) -> bool: ...
+    @property
+    def readable(self) -> bool: ...
+    @property
+    def writable(self) -> bool: ...
```

Comment:
Please add `# undocumented` comments to these properties. Alternatively (and probably preferably) you could file a bug report against Python's documentation in the [bug tracker](https://bugs.python.org/).

Reply:
Also, the `closed` property.